R/covariate_args.R

```diff
@@ -0,0 +1,13 @@
+#' Assigns common covariate arguments to argparse object
+#'
+#' @param parser argparse::ArgumentParser instance or equivalent
+covariate_args <- function(parser, ..., output_root = "<covariate root>") {
+  parser$add_argument("--lsid", default = 111, type = "integer", help = "Location set version id. Defaults to 111")
```

Comment:
Joe is writing the hierarchies out!  Can you read them in instead?

Reply:
Now done by reflecting values out of `model-inputs/best`